storage/integration_test.go

```diff
@@ -4335,7 +4407,7 @@ func TestIntegration_PostPolicyV4_SignedURL_WithSignBytes(t *testing.T) {
 		},
 	}
 
-	pv4, err := GenerateSignedPostPolicyV4(bucketName, objectName, ppv4Opts)
+	pv4, err := bucket.GenerateSignedPostPolicyV4(objectName, ppv4Opts)
```

Comment:
I don't think we have to switch for this test since it's explicitly about using the `SignRawBytes` option.

Reply:
makes sense 👍